example/src/repository/database/models/index.js

```diff
@@ -45,6 +46,11 @@ const operatorsAliases = {
   $col: Op.col,
 };
 
+if (process.env.AUDIT_LOG) {
```

Comment:
Doen't seem a very good approach, because if you put "AUDIT_LOG=false" in my env file this condition will be evaluated as true!

process.env.AUDIT_LOG == 'true' i think is the best!

Reply:
Fixed